project.clj

```diff
@@ -9,7 +9,8 @@
               "crux-kafka-connect"
               "crux-kafka"
               "crux-uberjar"
-              "crux-test"])
+              "crux-test"
+              "crux-metrics"])
```

Comment:
these are ordered - `lein sub install` (for example) is called on each of them in turn. `crux-test` should depend on `crux-metrics` (as will `crux-uberjar`, now I think about it) so we'll want to build `crux-metrics` before both of them